createdb.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+# TODO - it'll be trivial, but put this into a helper perhaps config.go and
+# call it any time we try to open the db and find it doesn't exist.
+# For now, run this script by hand to create the db
```

Comment:
We'll want that change done before 0.2 as otherwise we'd need to add a workaround to the packaging.